modules/imgimport/src/imgimport.cpp

```diff
@@ -29,17 +29,59 @@
     SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 */
 
-#include "imgimport.h"
+#include <opencv2/highgui/highgui.hpp>
+#include <opencv2/core/core.hpp>
+#include <iostream>
+#include "../../core/include/frame.h"
+#include "../../core/src/frame.cpp"
+#include "../include/imgimport.h"
 #include <vector>
+#include <string>
+#include "csvreading.cpp"
+#include <dirent.h>
```

Comment:
Note that this will break compiling the project with Microsoft Visual C++, but there are implementations of dirent available for MSVC which will have to be noted as a dependency for building with MSVC.